Changes

```diff
@@ -483,12 +483,12 @@ OCaml 5.4.0
   Xavier Leroy, Miod Vallat, Gabriel Scherer and Stephen Dolan, review by Miod
   Vallat, Vincent Laviron and Xavier Leroy)
 
-* #13050: Use '$' instead of '.' to separate module names in symbol names.
-  This changes mangling of OCaml identifiers from
+* #13050, #14104, #?????: Use '$' instead of '.' to separate module names
+  in symbol names on non Apple and non-unix operating systems.
```

Comment:
as far as I can tell, $ is used on apple and non-UNIX systems, so the "non apple" above should be "apple".

Reply:
Indeed, this is fixed now, thanks !